scripts/extensions.sh

```diff
@@ -57,7 +59,7 @@ echo -e "\n\n## meza: Install composer-supported extensions"
 cd "$m_mediawiki"
 cmd_profile "START extensions composer require"
 composer require \
-	mediawiki/semantic-media-wiki:~2.0 \
+	mediawiki/semantic-media-wiki:dev-master#787d70fb7f1c1db800e02e0dde0eee8ccc4b7e95 \
```

Comment:
I think this is for #362 but I can't tell what the problem is that this fixes or why we can't check out 2.4 or some other actual release.